telematic_system/telematic_apps/web_app/client/src/pages/AdminPage.js

```diff
@@ -159,7 +158,7 @@ const AdminPage = () => {
                 <UserRoleManagement
                     users={users}
                     orgs={orgs}
-                    orgsUsers={orgsUsers}
+                    orgsusers={orgsUsers}
```

Comment:
Should this within the {} be lowercase too? I see everywhere else that seems to have been changed to lowercase.

Reply:
This is left as it on purpose as it refers to the state variable const [orgsUsers, setOrgsUsers] = useState([]); The html attribute are recommended to use all lowercase due to code smell.